command/pr_create_test.go

```diff
@@ -50,11 +20,12 @@ func TestPRCreate(t *testing.T) {
 		} } } }
 	`))
 
-	origGitCommand := git.GitCommand
-	git.GitCommand = test.StubExecCommand("TestPrCreateHelperProcess", "clean")
-	defer func() {
-		git.GitCommand = origGitCommand
-	}()
+	cs, cmdTeardown := initCmdStubber()
+	defer cmdTeardown()
+
+	cs.Stub("")                                         // git status
+	cs.Stub("1234567890,commit 0\n2345678901,commit 1") // git log
+	cs.Stub("")                                         // git push
```

Comment:
I love the command stubber! One idea: instead of denoting the expected command in code comments, how about we enforce it in the expectation itself?

```suggestion
	cs.Stub("git status", "")
	cs.Stub("git log", "1234567890,commit 0\n2345678901,commit 1")
	cs.Stub("git push", "")
```

I think it would make for more resilient tests in the long run. I'm not sure how easy would it be to make it "match", though.

Reply:
this is a direction i thought we might go if you liked the `CmdStubber`. I'd love to add this; would you be ok with it being a follow-up?

Reply:
Yep, followup is fine! ⭐️